views/slices/ImageWithCaption.ejs

```diff
@@ -0,0 +1,28 @@
+<% 
+  const imageUrl = slice.primary.image.url; 
+  const caption = ctx.prismicH.asHTML(slice.primary.caption) 
+%>
+<div class="post-part single container">
+  <% if (slice.slice_label=='image-full-width'){%>
```

Comment:
1. Add a space before the closing EJS tag.
2. Add spaces before and after the `==` operator.
3. Change this to a triple equals operator: `===`.

Reply:
Made all changes.